src/FSharp.Editing.VisualStudio/ProjectSystem/ProjectProvider.fs

```diff
@@ -156,17 +156,18 @@ type internal ProjectProvider
                 cache := Some opts
                 return opts
         }
-
+    let projectDescriptor = 
+        lazy
+            { IsForStandaloneScript = false
+              ProjectFile           = projectFileName.Value
+              TargetFramework       = targetFramework.Value
+              CompilerVersion       = None // This option is only relevant in scripts
+              CompilerOptions       = compilerOptions.Value
+              SourceFiles           = sourceFiles.Value
+              FullOutputFilePath    = fullOutputPath.Value
```

Comment:
Why all the fields are lazy? IProjectProvider.Project is already lazy, so no point in more laziness here.